test/fork/integration/Ejection.t.sol

```diff
@@ -73,6 +73,16 @@ contract EjectionTest is Test, Utilities, DeploymentFixtures {
             emit IWithdrawalVault.WithdrawalRequestAdded(
                 _prepareWithdrawalRequestData(pubkeys[i])
             );
+            vm.expectCall(
+                address(csm),
+                abi.encodeWithSelector(
+                    IStakingModule.onValidatorExitTriggered.selector,
+                    nodeOperatorId,
+                    pubkeys[i],
+                    expectedFee,
+                    ejector.VOLUNTARY_EXIT_TYPE_ID()
```

Comment:
why does it work? this is a call and there should be an expectEmit check from lines above